pandas/io/excel/_odfreader.py

```diff
@@ -80,6 +87,16 @@ def get_sheet_data(self, sheet, convert_float: bool) -> List[List[Scalar]]:
         table: List[List[Scalar]] = []
 
         for i, sheet_row in enumerate(sheet_rows):
+
+            should_continue, should_break = self.should_skip_row(
```

Comment:
I am not sure this interface is obvious at all. prefer simple routines

if should_skip_rows(...):
....

when is a break condition? can you just do it here?

Reply:
I changed it. Break condition is now handled differently